swupd/create_manifests.go

```diff
@@ -429,15 +470,25 @@ func CreateManifests(version uint32, minVersion uint32, format uint, statedir st
 		return nil, err
 	}
 
-	// copy over unchanged manifests
-	addUnchangedManifests(&newMoM, oldMoM, groups)
-
 	// allManifests must include newManifests plus all old ones in the MoM.
 	allManifests, err := aggregateManifests(newManifests, &newMoM, version, c)
 	if err != nil {
 		return nil, err
 	}
 
+	// Create Iterative manifests if there isn't a format bump
+	if format == oldFormat {
+		var iManifests []*Manifest
+		iManifests, err = newMoM.writeIterativeManifests(newManifests, verOutput)
+		if err != nil {
+			return nil, err
+		}
+		newManifests = append(newManifests, iManifests...)
+	}
+
+	// copy over unchanged manifests
+	addUnchangedManifests(&newMoM, oldMoM, groups)
+
 	var osIdx *Manifest
 	if osIdx, err = writeIndexManifest(&c, &ui, allManifests); err != nil {
```

Comment:
@otaviobp Should the index manifest have an iterative manifest? Currently, a delta manifest for the index manifest is created, but an iterative manifest is not.

Reply:
This isn't going to break anything and could be added later as a future enhancement.